tests/frontend/settings-coderabbit.test.js

```
@@ -0,0 +1,214 @@
+      expect(spaRegex.test('/settings/billing')).toBe(true);
+      
+      // Should NOT match API routes
+      expect(spaRegex.test('/api/health')).toBe(false);
+      expect(spaRegex.test('/static/js/main.js')).toBe(false);
+      expect(spaRegex.test('/webhook/stripe')).toBe(false);
+    });
+  });
+});
+
+// Export for other tests to use
+export { mockSettings };
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)